babel/messages/extract.py

```diff
@@ -398,7 +398,13 @@ def extract_python(fileobj, keywords, comment_tags, options):
     in_def = in_translator_comments = False
     comment_tag = None
 
-    encoding = parse_encoding(fileobj) or options.get('encoding', 'iso-8859-1')
+    # In Python2, the default source encoding is US-ASCII
+    # In Python3, the default source encoding is UTF-8
+    # http://legacy.python.org/dev/peps/pep-0008/#source-file-encoding
+    default_encoding = 'US-ASCII' if PY2 else 'UTF-8'
```

Comment:
What will now happen when scanning a Python file that was encoded using latin-1 but without the encoding declared? Success if strings are ASCII, encoding error if something like 'ë' is encountered?